src/views/Portal/index.view.tsx

```diff
@@ -18,11 +19,12 @@ const PortalView: React.FC = () => {
           setToken("")
```

Comment:
```typescript
    if (user?.email_verified) {
...
```

This needs to be updated as well, I think?

Reply:
Since this is the portal view, only email verified users should be viewing/using it. Not 100% sure if it's necessary or if there will be any other way to access that view with a non-verified email but we can definitely remove it if you feel it's best to remove it.

Reply:
Ahh, it's fine to keep it this way then.

Reply:
Yeah we should keep it, I actually think unverified users can access the portal view. I just tried it out, I navigated to "portal" before signing in then I registered a new account, after authenticating I was redirected back to the portal view.

Reply:
unless we're removing the option to navigate to your portal on the main page before logging in

Reply:
@Alperdec is this what you're seeing? If so, this is the Verify page and should be expected. If not, we probably did something wrong. 

![image](https://user-images.githubusercontent.com/20177171/129297235-eb1258c7-cf07-453b-94e9-a21488b727b9.png)

Reply:
I also just tried replicating your method and was not able to. But the behavior that William commented is what I got and what we should be expecting. Based on web interactions on the final UI design, we can change this code in the future.

Reply:
Whoops. It's working as expected